### DISCUSSION TITLE
Podman Desktop init fails for RHEL 9

### DISCUSSION THREAD

**Discussion by nzineer** [podman-desktop/podman-desktop#9269] [Q&A]
Hello Friends,

Following [using RHEL WSL as a podman machine link](https://podman-desktop.io/blog/2024/08/01/using-rhel-wsl-podman-machine) I tried to setup a RHEL 9 machine.
Downloaded the tar.gz after installing all required packages from redhat.
Post download, the rhel9 machine does not spin up if user mode networking has been opted for.
Podman was installed using podman-desktop-airgap-1.12.0-setup-x64.exe for a restricted environment sitting behing a VPN and uses proxies. A user mode networking for fedora spins up just fine everytime, but it does not work for RHEL unless podman-net-usermode machine is pre-existing in WSL. 

Based on this [https://github.com/containers/podman/issues/24168](https://github.com/containers/podman/issues/24168 ) raised thereafter, I was keen to understand what can possibly be done to make RHEL machines run on Windows Pro/Enterprise?

Thanks in Advance.

**Comment by nzineer** [comment]
@jeffmaury Started a discussion here

**Reply by jeffmaury** [reply]
Maybe WSL [mirrored mode networking](https://learn.microsoft.com/en-us/windows/wsl/networking#mirrored-mode-networking) may help ?
